Subnets.tf

```diff
@@ -0,0 +1,110 @@
+
+resource "aws_subnet" "Geschäftsführung" {
+  vpc_id     = aws_vpc.opt-vpc.vpc_id
+  cidr_block = "10.0.0.0/8"
```

Comment:
why /8?
Do you think you need such a big network?
I would suggest /16. 
Otherwise it could be difficult to connect the vpc with other vpcs later